### PR TITLE
Devx 2491 snippets pagination

### PR DESCRIPTION
**Pull request by igorwojda**
## Description

- https://nexmoinc.atlassian.net/browse/DEVX-2491
- Extract info tutorial NOTE into partials

## Deploy Notes

https://nexmo-developer-pr-2755.herokuapp.com/client-sdk/in-app-messaging/guides/handling-pagination

### CODE REVIEW

File: _documentation/en/client-sdk/in-app-messaging/guides/handling-pagination.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+This guide covers the process loading and displaying chunks of conversation events at a time.
+
+```partial
+source: _partials/client-sdk/messaging/chat-app-tutorial-note.md
+```
+
+Given a Conversation you are already a Member of, you can download chunks (pages) of events existing within this conversation. 
+
+Load first page off events:
```
Comment (tbedford) [inline _documentation/en/client-sdk/in-app-messaging/guides/handling-pagination.md:16]: New line required before code block

Comment (tbedford) [inline _documentation/en/client-sdk/in-app-messaging/guides/handling-pagination.md:16]: "Load first page off events:" -> "Load first page of events:"?

Comment (igorwojda) [reply _documentation/en/client-sdk/in-app-messaging/guides/handling-pagination.md]: 077c6b0f9d97ef1759795b65783ae32e34a84bb0

Comment (igorwojda) [reply _documentation/en/client-sdk/in-app-messaging/guides/handling-pagination.md]: Fixed 2805b3326fde57b2ae9b341b59b9b4fdfdce74ad

**Pull request merged**